queries/typst/injections.scm

```diff
@@ -0,0 +1,7 @@
+(raw_blck
+  (blob) @injection.shebang @injection.content)
```

Comment:
We don't have `@injection.shebang`.

Reply:
Thank you for the feedback. The original author of the parser seems to be using that with helix and I just copied the queries from there. Maybe I should learn more about treesitter and maintain this myself.

Reply:
Yes, you cannot in general just copy the queries from the repository verbatim; they are more meant as a starting place.

(We are trying to make it easier to share queries across editors, but as some components are necessarily editor-specific -- like predicates, or editor-specific features like conceal -- having fully centralized queries is impossible for now.)